packages/cozy-device-helper/src/deeplink.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * This file is used to open the native app from a webapp
+ * if this native app is installed
+ *
+ * From a webapp, we don't have any clue if a native app is installed.
+ * The only way to know that, is to try to open the custom link
+ * (aka cozydrive://) and if nothing happens (no blur) we redirect to
+ * the callback
+ *
+ * Firefox tries to open custom link, so we need to create an iframe
+ * to detect if this is supported or not
+ */
+const _createHiddenIframe = (target, uri) => {
+  const iframe = document.createElement('iframe')
+  iframe.src = uri
+  iframe.id = 'hiddenIframe'
+  iframe.style.display = 'none'
+  target.appendChild(iframe)
+  return iframe
+}
+
+const openUriWithHiddenFrame = (uri, failCb) => {
+  const timeout = setTimeout(function() {
+    failCb()
+    handler && handler.remove()
```

Comment:
I didn't know `handler.remove` could be a thing? [The documentation says `addEventListener` returns nothing](https://developer.mozilla.org/fr/docs/Web/API/EventTarget/addEventListener). What's the difference with `window.removeEventListener('blur', onBlur)`?

Reply:
Old code, is old. Thank you. Fixed